### ISSUE TITLE
Problem with Lighting after update

### ISSUE THREAD

**Issue by LendaUrbana** [Kopernicus/Kopernicus#249]
KSP: 1.3.1.1891 Windows 10 64bit

Problem:  I reported an error in the mod SSRSSContinued (https://github.com/Galileo88/SSRSSContinued/issues/2) that happened after upgrading Kopernicus to 1.3.1-3. The spacecraft and space center are without light. I think I may have some use for you. Follow the print screens  https://imgur.com/a/FZIEx . I returned kopernicus to a previous version and the problem does not occur

Mods installed:

RealSolarSystem v12.0 for KSP 1.2.2
SSRSS v0.0.3.1 - SSRSSClouds_HighRes
Kopernicus 1.3.1-3
Sigma Dimensions v0.9.6
Scatterer 2.v0.0320b
Module Manager 3.00

Reproduction steps:

Install the version 1.3.1-3 of Kopernicus 

Log:

<link>https://www.dropbox.com/sh/s8ouv3hy8acb87q/AABp7CXFaxjp4i_FkWA9_COPa?dl=0

**Comment by Mrcarrot1** [comment]
@LendaUrbana https://forum.kerbalspaceprogram.com/index.php?/topic/83212-how-to-get-support-read-first/

**Comment by LendaUrbana** [comment]
Thank you so much. I did not know these rules. post edited

**Comment by Mrcarrot1** [comment]
RSS doesn't officially support 1.3, so you technically can't report bugs on it in 1.3.

**Comment by LendaUrbana** [comment]
I'll wait for the SSRSSContinued mod update or maybe Kopernicus. I'll use the previous version of Kopernicus. Should I close the question?

**Comment by Galileo88** [comment]
RSS doesn’t support it but SSRSS does. I made it compatible with 1.3-1.3.1.. but bug reports do not need to be made to the Kopernicus devs regarding other mods as they have nothing to do with planet packs except that they provide the framework. I saw your report on SSRSSs repo. I will fix it when I have time. Until then, remove the sun curve cfg. Things have changed in the latest version of kopernicus and many planet packs need adjustments, SSRSS being one of them. Have some patience and I’ll get a fix out soon.
